client/app/common/hero/hero.js

```diff
@@ -6,6 +6,8 @@ let heroModule = angular.module('hero', [
   uiRouter
 ])
 
-.component('hero', heroComponent);
+.component('hero', heroComponent)
+  
+.name;
 
 export default heroModule;
```

Comment:
So are you saying that it shouldn't be in the hero module or is this for all of the project and changes.

If it's specifically for the hero module, then I don't agree. I was trying to keep the code style consistent across all the modules. 

@toddmotto does the same thing here:

```
import angular from 'angular';
import uiRouter from 'angular-ui-router';
import CalendarComponent from './calendar.component';

const calendar = angular
  .module('calendar', [
    uiRouter
  ])
  .component('calendar', CalendarComponent)
  .config(($stateProvider, $urlRouterProvider) => {
    $stateProvider
      .state('calendar', {
        url: '/calendar',
        component: 'calendar'
      });
    $urlRouterProvider.otherwise('/');
  })
  .name;

export default calendar;
```

I've been basing most of my changes off of his style guide. I like this pattern.